setup.py

```diff
@@ -55,11 +55,12 @@
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
```

Comment:
This is removed but that's one of the versions we are testing.

Reply:
Oops, will fix!

Reply:
Fixed!